go/vt/worker/split_diff.go

```diff
@@ -219,15 +221,25 @@ func (sdw *SplitDiffWorker) findTargets(ctx context.Context) error {
 
 	// find an appropriate tablet in destination shard
 	var err error
-	sdw.destinationAlias, err = FindWorkerTablet(ctx, sdw.wr, sdw.cleaner, nil /* tsc */, sdw.cell, sdw.keyspace, sdw.shard, sdw.minHealthyRdonlyTablets)
+	sdw.destinationAlias, err = FindWorkerTablet(
+		ctx,
+		sdw.wr,
+		sdw.cleaner,
+		nil, /* tsc */
+		sdw.cell,
+		sdw.keyspace,
+		sdw.shard,
+		1, /* minHealthyTablets */
```

Comment:
This is also a behavior change worth calling out -- it seems like the appropriate change to me, but previously the split was as restrictive about how many healthy RDONLY tablets are in the destination as it is in the source.

Reply:
Agree with this change.